sqlglot/helper.py

```diff
@@ -125,3 +125,7 @@ def csv_reader(table):
         yield csv_.reader(file, delimiter=delimiter)
     finally:
         file.close()
+
+
+def invert(dictionary):
```

Comment:
this may not be necessary if you use the premade inverse_time stuff or self.format_time directly